deployments/cognito-rds-s3/terraform/versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.30.0"
+      version = ">= 5.0.0, < 6.0.0"
```

Comment:
will this override https://github.com/awslabs/kubeflow-manifests/blob/4e247e3722150163f99528880be46a30cf38bdbe/deployments/rds-s3/terraform/versions.tf#L6 ?

Reply:
Theres one for each deployment.
Heres what it looks like when running terraform init:
`Finding hashicorp/aws versions matching ">= 3.72.0, >= 4.10.0, >= 4.13.0, >= 4.30.0, >= 4.47.0, >= 5.0.0, < 6.0.0"...`